src/invidious/routes/misc.cr

```diff
@@ -35,4 +35,15 @@ class Invidious::Routes::Misc < Invidious::Routes::BaseRoute
     locale = LOCALES[env.get("preferences").as(Preferences).locale]?
     rendered "licenses"
   end
+
+  def redirect(env)
```

Comment:
Please change this to be generic, something like https://github.com/iv-org/invidious/blob/c9a316ad353cd1ee38c34d1aff0759765b37b2b3/src/invidious/views/template.ecr#L63